### PR TITLE
Adds a Fisanduhian ushanka to the loadout

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
For the record, this PR does not work! I don't know, it just doesn't compile. Likely because of
1. Me not understanding the faction system
or
2. Ushankas be straight quirky
I'll try to figure out my mistake, in the mean time.

Edit: NVM it was my BYOND version

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

**Comment by TheDanseMacabre** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -324,3 +324,8 @@
 	gadpathur["gadpathurian sidecap"] = /obj/item/clothing/head/gadpathur
 	gadpathur["gadpathurian beret"] = /obj/item/clothing/head/beret/gadpathur
 	gear_tweaks += new/datum/gear_tweak/path(gadpathur)
+
+/datum/gear/head/dominia
+	display_name = "patriotic ushanka"
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:329]: this needs to say which faction it's patriotic for

File: code/modules/clothing/factions/dominia.dm [hunk 2]
```diff
@@ -377,3 +377,8 @@
 	item_state = "dominia_amulet"
 	icon_state = "dominia_amulet"
 	contained_sprite = TRUE
+
+/obj/item/clothing/head/ushanka/dominia
+	name = "patriotic ushanka"
```
Comment (Geevies) [inline code/modules/clothing/factions/dominia.dm:382]: ditto

**Comment by NonQueueingMatt** [comment]
Changelog please.

**Review by fernerr — APPROVED** [review]

**Pull request merged**